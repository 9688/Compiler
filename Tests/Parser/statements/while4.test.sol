
WHILE
     list
          ">"
               1
               2
     BLOCK

