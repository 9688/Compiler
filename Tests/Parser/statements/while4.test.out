юдд''
   юдд'while'
      цдд'>'
      Ё  цдд'1'
      Ё  юдд'2'
      юдд'empty block'

